src/index.html

```diff
@@ -1,14 +1,101 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__img__box"></div>
+      <img
+          src="images/logo.png"
+          alt="Logo moyo"
+          class="header__img"
+       >
+      </div>
+
+      <nav class="nav">
+        <li class="header__nav__li">
+
+          <a href="#" class="header__nav__link is-active">
+
+            Apple
+
+          </a>
+
+        </li>
+
```

Comment:
Не залишай пустих рядків між тегами
```suggestion
        <li class="header__nav__li">
          <a href="#" class="header__nav__link is-active">
            Apple
          </a>
        </li>
```

Reply:
![image](https://user-images.githubusercontent.com/83292981/119817816-15fc7c00-bef7-11eb-9f18-7ac432e19faa.png)
блин я не правильно понял подумал что там пропуски есть
я ещё и думал зачем их делать)